src/databricks/labs/ucx/queries/progress/main/01_1_percentage_udf_migration_readiness.sql

```diff
@@ -0,0 +1,5 @@
+/* --title 'UDF migration readiness (%)' */
+SELECT
+    100 * COUNT_IF(size(failures) = 0) / COUNT(*) AS percentage
```

Comment:
```suggestion
    100 * COUNT_IF(SIZE(failures) = 0) / COUNT(*) AS percentage
```

nit: `SIZE()` is a function, hence should be uppercase